pkg/cmd/monitor_command/run_monitor_command.go

```diff
@@ -66,6 +67,13 @@ func NewRunMonitorCommand(ioStreams genericclioptions.IOStreams) *cobra.Command
 	cmd.Flags().StringVar(&monitorOpt.ArtifactDir,
 		"artifact-dir", monitorOpt.ArtifactDir,
 		"The directory where monitor events will be stored.")
+	cmd.Flags().StringVar(&monitorOpt.LoadBalancerType,
+		"lb-type", monitorOpt.LoadBalancerType,
+		"Set load balancer type, available options: internal-lb, service-network, external-lb (default)")
+	cmd.Flags().StringVar(&monitorOpt.ExtraMessage,
+		"extra-message", monitorOpt.ExtraMessage,
+		"Add custom label to disruption event message")
+	cmd.Flags().BoolVar(&monitorOpt.APIDisruptionOnly, "api-disruption-only", monitorOpt.APIDisruptionOnly, "Run api disruption monitoring only")
```

Comment:
This isn't an option we want.  I'm ok with you creating a disruption command above, but the monitor means, "everything" not "select my favorite things".

Reply:
This option is useful to avoid saving resources multiple times - once in disruption test container and on every pod where in-cluster monitor is running